### PR TITLE
Client+Server class additions - updated 5/15/17

### PR DESCRIPTION
**Pull request by ageannopoulos**
I made these classes more simplified but still demonstrative of JMS message interpretation, added the licenses, and cleaned up code. 

I also made the client generate messages that could have one of three messages to show that the server will reply to the correct client via the TemporaryQueue.

### CODE REVIEW

File: qpid-jms-examples/src/main/java/org/apache/qpid/jms/example/Server.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+import javax.jms.TextMessage;
+import javax.jms.ObjectMessage;
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import java.util.ArrayList;
+import java.util.Collections;
+
+public class Server {
+    public static void main(String[] args) throws Exception {
+	        try {
```
Comment (gemmellr) [inline qpid-jms-examples/src/main/java/org/apache/qpid/jms/example/Server.java]: The try block and its contents are indented an extra level than it should be. Your editor should be able to fix the indent. Feel free to grab me on IRC for assistance if needed.

File: qpid-jms-examples/src/main/java/org/apache/qpid/jms/example/Server.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+	            connection.setExceptionListener(new MyExceptionListener());
+	            connection.start();
+	
+	            Session session = connection.createSession(false, Session.AUTO_ACKNOWLEDGE);
+	
+	            MessageConsumer messageConsumer = session.createConsumer(queue);
+	            MessageProducer messageProducer = session.createProducer(null);
+	            
+	            while (true) {
+	                System.out.println("[SERVER] Awaiting Message...");
```
Comment (gemmellr) [inline qpid-jms-examples/src/main/java/org/apache/qpid/jms/example/Server.java]: The lines below this are indented an extra level, they should all be at this level. I also think we can simplify this section and improve the output a little by doing something like:

```
while (true) {
    TextMessage receivedMessage = (TextMessage) messageConsumer.receive();

    System.out.println("[SERVER] Received: " + receivedMessage.getText());

    TextMessage responseMessage = session.createTextMessage(receivedMessage.getText().toUpperCase());

    messageProducer.send(receivedMessage.getJMSReplyTo(), responseMessage, DeliveryMode.NON_PERSISTENT, Message.DEFAULT_PRIORITY, Message.DEFAULT_TIME_TO_LIVE);
}
```

**Comment by ageannopoulos** [comment]
Robbie,
Thanks for all the feedback. I'm very new to the open source world and your suggestions and advice is very helpful for me. I updated the classes again following the suggestions- hopefully they're close to being if not already finalized!

**Comment by gemmellr** [comment]
Hi Austin,

I've squashed your chagnes into a single commit (https://github.com/apache/qpid-jms/commit/72c7bb0880b5cc7dd332c0220091f93d0de9a5a2) and put it in along with further commit (https://github.com/apache/qpid-jms/commit/8a1a498c6c8307da1e3079c09c3d4d2e0aff5776) to fix up some remaining indentation+tab issues and make some small simplification+clarification changes. Thanks for the contribution!

**Pull request closed without merging**